src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -420,6 +420,17 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 6.30951909821307.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1897
1897:             $pathologicalRatio | Should -BeGreaterThan 10
```